java/src/main/java/org/eclipse/ditto/client/live/internal/LiveThingHandleImpl.java

```diff
@@ -103,40 +104,32 @@ protected LiveFeatureHandleImpl createFeatureHandle(final ThingId thingId, final
             final Class<T> type,
             final Consumer<RepliableMessage<T, U>> handler) {
 
-        argumentNotNull(registrationId, "registrationId");
-        argumentNotNull(subject, "subject");
-        argumentNotNull(type, "type");
-        argumentNotNull(handler, "handler");
-
+        LiveMessagesUtil.checkSubject(subject);
         LiveMessagesUtil.checkSerializerExistForMessageType(messageSerializerRegistry, type, subject);
 
         // selector for thing messages:
         final JsonPointerSelector thingSelector = "*".equals(subject)
                 ? SelectorUtil.formatJsonPointer(LOGGER, "/things/{0}/'{direction}'/messages/'{subject}'",
                 getEntityId())
-                :
-                SelectorUtil.formatJsonPointer(LOGGER, "/things/{0}/'{direction}'/messages/{1}", getEntityId(),
-                        subject);
+                : SelectorUtil.formatJsonPointer(LOGGER, "/things/{0}/'{direction}'/messages/{1}",
+                getEntityId(), subject);
         // selector for feature messages:
         final JsonPointerSelector featureSelector = "*".equals(subject)
                 ? SelectorUtil.formatJsonPointer(LOGGER,
                 "/things/{0}/features/'{featureId}'/'{direction}'/messages/'{subject}'", getEntityId())
-                :
-                SelectorUtil.formatJsonPointer(LOGGER, "/things/{0}/features/'{featureId}'/'{direction}'/messages/{0}",
-                        getEntityId(), subject);
+                : SelectorUtil.formatJsonPointer(LOGGER,
+                "/things/{0}/features/'{featureId}'/'{direction}'/messages/{1}", getEntityId(), subject);
```

Comment:
@stmaute that's the actual fix? 🤣

Reply:
Yes that is it. The entityId was used in the selector instead of the subject.